command-service/src/config/service_config.yml

```diff
@@ -15,6 +15,7 @@ flink:
 commands:
   PUBLISH_DATASET:
     workflow:
+      - CREATE_KAFKA_TOPIC
```

Comment:
@JeraldJF Are you doing a validation if the dataset is already Live before calling this command?

Reply:
Yes @anandp504